pebble/internal/validator.go

```diff
@@ -0,0 +1,21 @@
+package internal
```

Comment:
Do we really need a full file for 1 function?

Reply:
I put that just to keep things separated. But I can also move it into `pebble.go`.